Grammar (#76)
fhdsl/NIH_Data_Sharing: 02-data_management_plan.Rmd

<div class = "question">
Where will I share my data? What repositories exist that might work for my data type?
</div>

> Some programs or Funding Opportunity Announcements (FOA) will specify where the data should be shared, in which case be sure to plan to use the correct repositories for the specific FOA you're responding to. 

Reviewer: ```suggestion
Some programs or Funding Opportunity Announcements (FOA) will specify where the data should be shared. If this applies, you should plan to use the repositories mentioned in the FOA. 
```

Author: I Like it!
